### PR TITLE
[AIRFLOW-6266] Better asserts in test_views.py

### PR DESCRIPTION
**Pull request by potiuk**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-6266

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

**Review by kaxil — APPROVED** [review]

File: tests/www/test_views.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def clear_table(cls, model):
 
     def check_content_in_response(self, text, resp, resp_code=200):
         resp_html = resp.data.decode('utf-8')
-        self.assertEqual(resp_code, resp.status_code)
+        assert resp_code == resp.status_code, str(resp_html)
```
Comment (mik-laj) [inline tests/www/test_views.py:98]: Can you use assertions from [unittest](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual) package?  Currently, this convention is used throughout the entire project. Keeping one style in the whole project facilitates its development.

**Review by mik-laj — CHANGES_REQUESTED** [review]
Please stick to the project convention. We should introduce new ideas, but if they relate to a significant part of the code, they should be accepted by the whole community.

**Comment by turbaszek** [comment]
@potiuk @mik-laj should we close or merge?

**Review by potiuk — COMMENTED** [review]
I am all for merging now :) @mik-laj ?

**Review by mik-laj — APPROVED** [review]

**Comment by mik-laj** [comment]
I preferred that we still use the old style of assertion, but if others think otherwise then I will not block. @nuclearpinguin WDYT?

File: tests/www/test_views.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def check_content_in_response(self, text, resp, resp_code=200):
 
     def check_content_not_in_response(self, text, resp, resp_code=200):
         resp_html = resp.data.decode('utf-8')
-        self.assertEqual(resp_code, resp.status_code)
+        assert resp_code == resp.status_code, str(resp_html)
```
Comment (turbaszek) [inline tests/www/test_views.py:107]:
```suggestion
     self.assertEqual(resp_code, resp.status_code, msg=str(resp_html)
```

File: tests/www/test_views.py (hunk 1 above)
Comment (turbaszek) [inline tests/www/test_views.py:98]:
```suggestion
    self.assertEqual(resp_code, resp.status_code, msg=str(resp_html)
```

**Review by turbaszek — COMMENTED** [review]
I think we can achieve the same result using current framework.

**Comment by turbaszek** [comment]
But to be honest, I see no value in this change. The message will be a big, purrrrre html 🙀

**Comment by potiuk** [comment]
But it's only when the response is wrong . We will see exactly what was returned - I already realised the problem after looking at the response what was the root cause.

**Pull request closed without merging**